token/cli/src/main.rs

```diff
@@ -417,6 +419,13 @@ async fn command_create_token(
         })
     }
 
+    if let Some(state) = default_account_state {
+        if !enable_freeze {
+            return Err("Token requires a freeze authority to default to frozen accounts".into());
```

Comment:
Seems like this is superfluous with `.requires("enable_freeze")` set? Should we assert/expect?

Reply:
I was thinking in case we call this function in a test incorrectly, but we never do that, so changed it to an assert. thanks!